src/views/GeneralLedger/GeneralLedger.tsx

```diff
@@ -25,13 +27,15 @@ export interface GeneralLedgerProps {
   showTitle?: boolean
   stringOverrides?: GeneralLedgerStringOverrides
   chartOfAccountsOptions?: ChartOfAccountsOptions
+  convertLedgerEntrySourceToSourceLink?: (source: LedgerEntrySource) => SourceLink | undefined
```

Comment:
Thinking about naming here - I want to specify 1) the shape of the function that needs to be passed, and 2) the result that it will have (i.e., showing a link). Something like: 
```suggestion
  withLedgerEntrySourceLink?: (source: LedgerEntrySource) => SourceLink | undefined
```